docs/configuration/app.md

```diff
@@ -57,14 +57,32 @@ In general, do not include any quotation marks when adding configuration values.
 
 ### Authentication
 
-#### `DISPATCH_AUTHENTICATION_PROVIDER` \['default': dispatch-auth-provider-pkce\]
+#### `DISPATCH_AUTHENTICATION_PROVIDER` \['default': dispatch-auth-provider-basic\]
 
 > Used by Dispatch to determine which authentication provider to use, by default Dispatch ships with a PKCE authentication provider.
 
 {% hint style="info" %}
 If you wish to disabled authentication set `DISPATCH_AUTHENTICATION_PROVIDER=""`
```

Comment:
Is this still valid? I don't see `DISPATCH_AUTHENTICATION_PROVIDER` being used anywhere in the code.

Reply:
Ah, should have `slug` added to the end. Will add.